src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -74,6 +81,14 @@ static jboolean initOnUncaught = JNI_FALSE; /* init when uncaught exc thrown */
 
 static char *launchOnInit = NULL;           /* launch this app during init */
 static jboolean suspendOnInit = JNI_TRUE;   /* suspend all app threads after init */
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+/* By default VM is suspended post-restore unless suspendOnRestore=n is specified.
+ * -1 initial value
+ *  0 not suspend post-restore
+ *  1 suspend post-restore
+ */
+static jbyte suspendOnRestore = -1;
```

Comment:
Suggest this continue to have type `jboolean` (else the use of `debugInit_suspendOnRestore()` is incorrect).
A local (boolean) variable in `parseOptions()` can be used to help distinguish these cases.

Reply:
Added a local `jboolean suspendOnRestoreTmp = JNI_TRUE`, this is to be updated according to `suspendOnRestore=[y|n]`, and assigned to the global `suspendOnRestore`.